shell/platform/common/client_wrapper/include/flutter/basic_message_channel.h

```diff
@@ -101,6 +102,41 @@ class BasicMessageChannel {
     messenger_->SetMessageHandler(name_, std::move(binary_handler));
   }
 
+  // Adjusts the number of messages that will get buffered when sending messages
+  // to channels that aren't fully set up yet. For example, the engine isn't
+  // running yet or the channel's message handler isn't set up on the Dart side
+  // yet.
+  //
+  // |new_size] is an int because the deserialization logic handles only 32 bits
```

Comment:
`int` isn't required to be 32-bit, so if the goal is to constrain it via types it should be an `int32_t`. Although I would probably just remove this comment and truncate to `int32_t` internally; we could always change the implementation if anyone ever actually needed a bigger values in the future, so clients probably don't need to know this.